### PR TITLE
Further reduce bundle size by code splitting out pages that are only used on desktop/mobile

### PR DESCRIPTION
**Pull request by j-f1**
Since people on desktops generally won’t be using the mobile page components, and vice versa, I’ve updated the main router to dynamically load in which ever one is necessary. Since all components are bundled together in one file per size class, the latency when switching routes is ~4ms. This should significantly decrease the bundle size on mobile, which is important since mobile devices tend to have lower specs.

Latency of opening `/budget` (measured from `NarrowAlternate` mounting to `BudgetWrapper` mounting) is 60ms locally, 140ms on Netlify with my WiFi, ~1s on the “Fast 3G” network speed preset)

Additionally, I discovered yet another font file we bundle but don’t use.

### CODE REVIEW

File: packages/desktop-client/src/components/schedules/index.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { Page } from '../Page';
 import { SchedulesTable, ROW_HEIGHT } from './SchedulesTable';
 
 export default function Schedules() {
+  console.log(`<Schedules> mounted at ${new Date().toISOString()}`);
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/schedules/index.js:13]: **❓ question:** ‏should this be removed?

File: packages/desktop-client/src/components/budget/index.js [hunk 2]
```diff
@@ -517,6 +517,7 @@ const RolloverBudgetSummary = memo(props => {
 });
 
 function BudgetWrapper(props) {
+  console.log(`<BudgetWrapper> mounted at ${new Date().toISOString()}`);
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/budget/index.js:520]: **❓ question:** ‏should this be removed too?

File: packages/desktop-client/src/components/responsive/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import type * as WideComponents from './wide';
+
+let loadNarrow = () =>
+  import(/* webpackChunkName: "narrow-components" */ './narrow');
+let loadWide = () => import(/* webpackChunkName: "wide-components" */ './wide');
+
+export function WideComponent({ name }: { name: keyof typeof WideComponents }) {
+  console.log(
+    `<WideComponent name=${name} /> mounted at ${new Date().toISOString()}`,
+  );
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/responsive/index.tsx:14]: also the console.logs in this file

**Review by MatissJanis — COMMENTED** [review]
Overall LGTM. Just the `console.log` I think we should clean up.

**Review by MatissJanis — APPROVED** [review]

**Review by MatissJanis — APPROVED** [review]

**Review by MatissJanis — CHANGES_REQUESTED** [review]
Actually: small bug.

1. Open the mobile version
2. Open an account
3. Click "back" button in the app
4. The app crashes

```
TypeError: Cannot read properties of undefined (reading 'id')
```

**Comment by j-f1** [comment]
Fixed!

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**